data/sql/updates/pending_db_world/rev_1582996192436902600.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1582996192436902600');
+
+-- Update gossip_menu table to allow bigger MenuIDs
+ALTER TABLE `gossip_menu` CHANGE COLUMN `MenuID` `MenuID` INT UNSIGNED NOT NULL DEFAULT 0 FIRST;
+
+-- Zone Stormwind - npc_lady_katrana_prestor gossip menu
+UPDATE npc_text SET TEXT0_0 = '<Lady Prestor glares at you.>', BroadcastTextID0 = '0' WHERE ID = '2693';
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES ('90000', '2693');
```

Comment:
Before insert you should add a DELETE.

Reply:
Is it really needed in this case? I picked a range that is currently not being used.

Reply:
It is always needed. Otherwise, you can only run the query once.